### DISCUSSION TITLE
EFCore SQLServer extensions

### DISCUSSION THREAD

**Discussion by alunacjones** [rebus-org/Rebus#1106] [Ideas]
Hi,

Having used rebus for a while now and wanting to integrate the SqlServer support with EfCore in my current job, I've created extensions for modelBuilder in a nuget package and was wondering if it would be useful to a wider audience and perhaps reviewed by yourselves as potentially being absorbed into the wider rebus set of packages.

The nuget package is [here](https://www.nuget.org/packages/LSL.Rebus.EfCore.SqlServer) and the repository is [here](https://github.com/alunacjones/LSL.Rebus.EfCore.SqlServer)

P.S. I feel the documentation should probably show examples for setting up via the Rebus.SqlServer package too 😄

**Comment by mookid8000** [comment]
Interesting! 🙂 I'm not that much of an EF Core expert – what exactly does the NuGet package do?
